config/deploy.rb

```diff
@@ -18,6 +18,9 @@
 
 set :bundle_jobs, 4
 
+# For Container Default Value
+set :image_version, ENV['VERSION'] || "latest"
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)